### ISSUE TITLE
Episode schedule

### ISSUE THREAD

**Issue by Jenrykster** [Jenrykster/animeo#5]
Stremio has a calendar system which shows when a new episode is airing (it also shows new episodes at the home screen). I want to look into making this addon integrate with the calendar.

**Comment by aliyss** [comment]
https://cinemeta-live.strem.io/meta/series/tt8910922/5/8.json

No clue... this might be some metadata that may help... like videos.0.released

Also you might want to look into manifest stuff like catalog.0.extra no clue if that makes a difference

**Comment by Jenrykster** [comment]
Yeah... now I'm wondering if I can even provide this kind of metadata if stremio is using what is provided by the kitsu addon, I'll try looking into it today, thanks for the resource linked.

**Comment by aliyss** [comment]
```js
  meta: {
        trailers: (animeInfo.trailers || []).map((t) => {
          return {
            source: t.youtube,
            type: "Trailer",
          };
        })
  }
```

Just in case... it's the trailers key

Edit: nvm... forgot you were looking for calendar not trailers
